indexing-service/src/main/resources/indexer_static/js/console-0.0.1.js

```diff
@@ -2,39 +2,56 @@
 
 var oTable = [];
 
+var killTask = function(taskId) {
+  if(confirm('Do you really want to kill: '+taskId)) {
+    $.ajax({
+      type:'POST',
+      url: '/druid/indexer/v1/task/'+ taskId +'/shutdown',
+      data: ''
+    }).done(function(data) {
+      alert('Submitted kill request for: '+data.task);
```

Comment:
coordinator also has a place to see indexing tasks. should this be added there as well or just there alone? i'm not sure what the right answer is but just mentioning this.